packages/features/ee/workflows/components/WorkflowListPage.tsx

```diff
@@ -157,6 +156,13 @@ export default function WorkflowListPage({ workflows }: Props) {
                               setwWorkflowToDeleteId(workflow.id);
                             }}
                             color="secondary"
+                            disabled={
+                              !!workflow.team?.members?.find(
```

Comment:
nit: I think it would be better to send isReadOnly from trpc response itself. Component should have as less logic as possible I think

Reply:
Changed that now. And I am also showing the readonly badge like we do for event types